BUILD

```diff
@@ -77,7 +77,31 @@ linux_packages(
     depends = [
         "openjdk-8-jre"
     ],
-    modes = {
+    permissions = {
+        "./var/log/grakn/": "0777",
+    },
+    symlinks = {
+        "./usr/local/bin/grakn": "/opt/grakn/core/grakn",
```

Comment:
here as well, `usr/...` instead of `./usr/...`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c